test_conformance/common/vulkan_wrapper/opencl_vulkan_wrapper.cpp

```diff
@@ -79,6 +80,15 @@ void init_cl_vk_ext(cl_platform_id opencl_platform)
         throw std::runtime_error("Failed to get the function pointer of "
                                  "clGetSemaphoreHandleForTypeKHRptr!");
     }
+
+    clReImportSemaphoreSyncFdKHRptr = (pfnclReImportSemaphoreSyncFdKHR)
```

Comment:
We should check if cl_khr_external_semaphore_sync_fd extension is supported or not before querying/calling this API.

Reply:
That sounds correct.  I will make the change.